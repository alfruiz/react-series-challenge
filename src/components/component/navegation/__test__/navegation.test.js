import React from 'react';
import { shallow } from 'enzyme';
import { NavLink } from "react-router-dom";

import Navegation from "./../index";

describe('<Navegation> ', () => {

  let wraper;

  beforeAll(() => {
    wraper = shallow(
      <Navegation />
    );
  });

  it('should have two (2) NavLinks', () => {
    expect(wraper.find(NavLink).length).toEqual(2);
  });
});