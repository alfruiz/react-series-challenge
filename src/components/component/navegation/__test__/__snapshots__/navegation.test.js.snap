// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navegation>  create a Snapshot  1`] = `
<header>
  <ul>
    <li>
      <Styled(NavLink)
        activeClassName="active"
        to="/home"
      >
        HOME
      </Styled(NavLink)>
    </li>
    <li>
      <Styled(NavLink)
        activeClassName="active"
        to="/keeps"
      >
        Favoritos
      </Styled(NavLink)>
    </li>
  </ul>
</header>
`;
